ArchUnitNET/Fluent/Syntax/ActivatorHandler.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using ArchUnitNET.Domain;
+
+namespace ArchUnitNET.Fluent.Syntax
+{
+    public static class ActivatorHandler
```

Comment:
ConjunctionFactory is a better name for it

Reply:
renamed it in new version